### DISCUSSION TITLE
How can I use next-translate with SSG?

### DISCUSSION THREAD

**Discussion by skb1129** [vinissimus/next-translate#700] [Q&A]
I want to static export my next application that is using `next-translate`. Currently, I get an error saying: `Error: i18n support is not compatible with next export.`. I want to use translation while being able to statically export the application as HTML pages.

**Comment by aralroca** [comment]
For now, you can not, unless you use version 0.18 of Next-translate. In 0.19 we changed the i18n routing (that we managed with workarounds) for the [i18n routing that Next.js released in its core](https://nextjs.org/docs/advanced-features/i18n-routing) in version 10.

So it's a feature that Next.js has to implement because all the [i18n routing](https://nextjs.org/docs/advanced-features/i18n-routing) is part of the core of Next.js and not of this library, this library now is only used for the translations of each page.
